kalite/settings/__init__.py

```diff
@@ -32,22 +34,24 @@ def package_selected(package_name):
 # IMPORTANT: Do not add new settings below this line
 #
 # Everything that follows is overriding default settings, depending on CONFIG_PACKAGE
-
-# config_package (None|RPi) alters some defaults e.g. different defaults for Raspberry Pi(RPi)
-# autodetect if this is a Raspberry Pi-type device, and auto-set the config_package
-#  to override the auto-detection, set CONFIG_PACKAGE=None in the local_settings
 ########################
 
-CONFIG_PACKAGE = getattr(local_settings, "CONFIG_PACKAGE", "RPi" if (platform.uname()[0] == "Linux" and platform.uname()[4] == "armv6l") else [])
-
+# A deprecated setting that shouldn't be used
+CONFIG_PACKAGE = CONFIG_PACKAGE = getattr(local_settings, "CONFIG_PACKAGE", [])
 if isinstance(CONFIG_PACKAGE, basestring):
     CONFIG_PACKAGE = [CONFIG_PACKAGE]
 CONFIG_PACKAGE = [cp.lower() for cp in CONFIG_PACKAGE]
 
 
+if CONFIG_PACKAGE:
+    warnings.warn(
+        "CONFIG_PACKAGE is outdated, see docs (TODO MCGallaspy, URL ?)",
```

Comment:
The URL for the docs -- sorry about tagging you, it was under the assumption that you were picking up after.. I was finding it difficult to link to a chapter in the docs because I didn't know the final location of them.